MainWindow.xaml.cs

```diff
@@ -105,26 +101,23 @@ private void cmb_DriveSpeed_SelectionChanged(object sender, SelectionChangedEven
             EnsureDiscInformation();
         }
 
-        private void tbr_Properties_Click(object sender, RoutedEventArgs e)
+        private void tbr_Options_Click(object sender, RoutedEventArgs e)
         {
-            ShowSettings();
-        }
+            // lazy initialization
+            if (_optionsWindow == null)
+                _optionsWindow = new OptionsWindow(_options);
 
-        private void tbr_Properties_CanExecute(object sender, System.Windows.Input.CanExecuteRoutedEventArgs e)
-        {
-            e.CanExecute = true;
-        }
+            //_optionsWindow.Owner = this;
+            //_optionsWindow.WindowStartupLocation = WindowStartupLocation.CenterOwner;
```

Comment:
Why are these two lines commented out? They seem like pretty valid options to enable.

Reply:
I commented them because they didn't work as expected but probably the problem was somewhere else.